packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -31,6 +31,8 @@ export const onClientEntry = (
         whitelistUrls: ['docs.commercetools.com', 'now.sh'],
       });
     });
+    if (window && !window.location.host.includes('.commercetools.com'))
+      document.body.classList.add('not-on-cookie-domain');
```

Comment:
Trying to be explicit in the name - that's what it really does, it's not actually "Production" or "staging". 

"window" boolean check intends to make sure that it does not run server side. 

using the not completely backwards compatible classList()  call  is okay since it's only called when not called on our production domain, which is ourselves looking at previews.